src/components/notifications/AppSelector/index.tsx

```diff
@@ -124,7 +126,7 @@ const AppSelector: React.FC = () => {
           )}
         </div>
         <div className="AppSelector__wrapper">
-          <Label color="main">Subscribed</Label>
+          {!empty ? <Label color="main">Subscribed</Label> : null}
```

Comment:
```suggestion
          {!empty && <Label color="main">Subscribed</Label>}
```

Reply:
doesn't this sometimes cause "false" or something to be in the markup

Reply:
This depends on what the `empty`'s type and value is. Since it's value checking both loading and filteredApp's length, it'll be boolean every time, and `&&` will work as expected. 

Yet, we should still consider following better approaches while doing conditional renderings. Developers shouldn't need to watch the value of the `empty`'s value manually. Instead, we should use the ternary operator. 

Please check this: https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

Reply:
Yes I'm aware of that "rule" but it's bloating the code and if the condition is based on a boolean you can always avoid using a ternary in a reliable way.

Up to your preferences I guess.